sites/shared/components/workbench/views/measies/editor.mjs

```diff
@@ -0,0 +1,27 @@
+import { MeasieInput, ns as inputNs } from 'shared/components/sets/measie-input.mjs'
+import { useTranslation } from 'next-i18next'
+
+export const ns = ['wbmeasies', ...inputNs]
+const emptyFunc = () => {}
```

Comment:
See remark about making `startLoading` and `stopLoading` default to this in `MeasieInput`